save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/projects/ProjectDemoMenu.kt

```diff
@@ -95,12 +94,12 @@ val projectDemoMenu: FC<ProjectDemoMenuProps> = FC { props ->
             setGithubProjectCoordinates(demoInfo.demoDto.githubProjectCoordinates.orEmpty())
         } else if (infoResponse.status != 404.toShort()) {
             props.updateErrorMessage(infoResponse.statusText, infoResponse.unpackMessage())
-            setDemoStatus(DemoStatus.NOT_CREATED)
+            setDemoStatus(DemoStatus.ERROR)
         }
     }
 
     useOnce {
-        window.alert("This is just a preview, nothing on this view works for now.")
+        // window.alert("This is just a preview, nothing on this view works for now.")
```

Comment:
a comment code

Reply:
Oops, shouldn't be commented, will fix it

Reply:
Uncommented the alert